Src/FluentAssertions/Primitives/EnumAssertions.cs

```diff
@@ -129,6 +129,52 @@ public AndConstraint<TAssertions> Be(TEnum? expected, string because = "", param
             return new AndConstraint<TAssertions>((TAssertions)this);
         }
 
+        /// <summary>
+        /// Asserts that the current value of <typeparamref name="TEnum"/> is defined inside the enum.
+        /// </summary>
+        /// <param name="because">
+        /// A formatted phrase as is supported by <see cref="string.Format(string,object[])" /> explaining why the assertion
+        /// is needed. If the phrase does not start with the word <i>because</i>, it is prepended automatically.
+        /// </param>
+        /// <param name="becauseArgs">
+        /// Zero or more objects to format using the placeholders in <paramref name="because" />.
+        /// </param>
+        public AndConstraint<TAssertions> BeDefined(string because = "", params object[] becauseArgs)
+        {
+            Execute.Assertion
+                .BecauseOf(because, becauseArgs)
+                .WithExpectation("Expected {context:the enum} to be defined in {0}{reason}, ", typeof(TEnum))
```

Comment:
When using `WithExpectation` you must end the chain of calls with

```cs
.Then
.ClearExpectation();
```

If you've found a place where we don't do it, that's probably a mistake.

Reply:
Got it :)

Reply:
Just another question about that `ClearExpectation`: Does this apply also to test cases where `WithExpectation` is used?

Reply:
I would assume, but it's not as crucial.
It would have to looked into case-by-case, as it might be testing a test where `ClearExpectation` is _not_ called.